src/components/FooterWrapper.js

```diff
@@ -78,6 +78,13 @@ const FooterWrapper = ({ children, alwaysShow, fixedHeight }) => {
     }, [
       'Documentation', icon('pop-out', { size: 12, style: { marginLeft: '0.5rem' } })
     ]),
+    div({ style: styles.item }, '|'),
```

Comment:
@marctalbott can you refactor this code to loop over the similar items to remove the duplication? I am thinking of an "item" as the separator div followed by the anchor tag with popout styling (Security, Documentation, Terra.bio).